target/linux/ath79/dts/qca9531_glinet_gl-s200.dtsi

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "glinet,gl-s200", "qca,qca9531";
+	model = "GL.iNet GL-S200";
+
+	aliases {
+		label-mac-device = &eth0;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
```

Comment:
I would prefer `gpio-keys` if it works. The driver will poll the GPIOs when you use `gpio-keys-polled` and it will use IRQs when you use `gpio-keys`. I prefer using IRQs to reduce load on the system. 

When you change this please check if it really works.

Reply:
I have modified it and checked if it works